client/src/components/layout/UserShow.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import UserSerializer from "../../../../server/src/serializers/UserSerializer";
+import BuildsList from "./BuildsList";
+import ReviewsList from "./ReviewsList"
+
+const UserShow = ({user}) => {
+    
+    const [currentUserData, setUserData] = useState({
+        buildsList: [],
+        reviewsList: []
+    })
+
+    const userInfo = UserSerializer.serializeUser(user)
+    const createdAt = new Date(userInfo.createdAt)
```

Comment:
this is LITERALLY hardcoding!! you have a createdAt in the DB, make a fetch for it if you need it! 🤦 But the this data is already in `App` anyways